identity-manager/src/config.rs

```diff
@@ -0,0 +1,112 @@
+use serde::Deserialize;
+use serde::Serialize;
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct Config {
+    pub trust_domain: String,
+    pub server_socket_path: String,
+    pub workload_attestation_plugin: String,
+    pub node_attestation_plugin: AttestationPlugin,
+    pub provisionning: Provisionning,
```

Comment:
```suggestion
    pub provisioning: Provisioning,
```

Also replace other occurences.